scipy/stats/_continuous_distns.py

```diff
@@ -781,13 +787,47 @@ def _pdf(self, x, c, d):
         # burr.pdf(x, c, d) = c * d * x**(-c-1) * (1+x**(-c))**(-d-1)
         return c * d * (x**(-c - 1.0)) * ((1 + x**(-c))**(-d - 1.0))
 
+    def _logpdf(self, x, c, d):
+        return np.log(c) + np.log(d) + (-c - 1.0) * np.log(x) \
+                + (-d - 1.0) * np.log1p(x**(-c))
+
     def _cdf(self, x, c, d):
         return (1 + x**(-c))**(-d)
 
+    def _logcdf(self, x, c, d):
+        return -d * np.log1p(x**(-c))
+
+    def _sf(self, x, c, d):
+        return 1 - (1 + x**(-c))**(-d)
+
     def _ppf(self, q, c, d):
         return (q**(-1.0/d) - 1)**(-1.0/c)
 
+    def _stats(self, c, d):
+        e = lambda k: sc.beta(d + k / c, 1.0 - k / c) / sc.beta(d, 1.0)
+        e1 = e(1.0)
+        e2 = e(2.0)
+        e3 = e(3.0)
+        e4 = e(4.0)
```

Comment:
Nit: sc.beta(d + np.arange(1, 5)/c, 1. - np.arange(1, 5)/c)`

Reply:
in most recent commit.